packages/components/ovh-shell/src/client/api.ts

```diff
@@ -136,6 +136,27 @@ export default function exposeApi(shellClient: ShellClient) {
           method: 'onChatbotClose',
           args: [reduced],
         }),
+      openLiveChat: () =>
+        shellClient.invokePluginMethod<void>({
+          plugin: 'ux',
+          method: 'openLiveChat',
+        }),
+      openChatbot: () =>
+        shellClient.invokePluginMethod<void>({
+          plugin: 'ux',
+          method: 'openChatbot',
+        }),
+      closeChatbot: () =>
+        shellClient.invokePluginMethod<void>({
+          plugin: 'ux',
+          method: 'closeChatbot',
+        }),
+      onChatbotVisibilityChange: (callback: CallableFunction) =>
```

Comment:
you need to remove this one, i think onChatbotVisibilityChange can only be called from the internal shell plugin and not be exposed in the client API because "callback" isn't serializable and won't be sent over iframe message bus